core/src/components/input/input.ionic.scss

```diff
@@ -33,7 +33,7 @@
   color: #{$ionic-color-neutral-500};
 }
 
-.input-clear-icon.ion-focused {
+.input-clear-icon:focus-visible {
```

Comment:
With this change should we remove the `ion-focusable` class from the JSX mark-up on the button?

Reply:
Tested and we don't need the class. It's been removed: https://github.com/ionic-team/ionic-framework/pull/29409/commits/6e49cb352314d525726269d408458d90227e7322